extensions/amp-story-panning-media/0.1/amp-story-panning-media.js

```diff
@@ -40,27 +43,30 @@ export class AmpStoryPanningMedia extends AMP.BaseElement {
     /** @private {?Element} The element that is transitioned. */
     this.ampImgEl_ = null;
 
-    /** @private {?string} Sent to siblings to update their position. */
+    /** @private {?number} */
     this.x_ = null;
 
-    /** @private {?string} Sent to siblings to update their position. */
+    /** @private {?number} */
     this.y_ = null;
 
-    /** @private {?string} Sent to siblings to update their position. */
+    /** @private {?number} */
```

Comment:
nit: can you comment what the expected/legal values are here?  0.0 to 1.0 inclusive?

Reply:
I did my best to explain it but any float value is legal. I think there are use cases for extremes on either end.

Zooming in really far, like 50 could create an interesting pixelated effect for a guessing game of what an image is. That concept could be combined with the quiz component.
Zooming from 0.0 would have an effect of the image appearing from a distance.